src/fixtures.js

```diff
@@ -1,4 +1,4 @@
-import {COLDCARD_BASE_BIP32, WALLET_CONFIG_VERSION} from './coldcard';
```

Comment:
Use shared fixtures from `unchained-bitcoin`.  Where not possible (e.g. Coldcard config format), store it in `coldcard.fixtures.js`.